package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/aio-lib-console",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
can you remove this edit, our publish system will auto-increment the patch version

Reply:
@shazron , done